code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -97,8 +100,9 @@
 			armed = FALSE
 			update_icon()
 			playsound(src.loc, 'sound/effects/snap.ogg', 50, 1)
-			L.visible_message("<span class='danger'>[L] triggers \the [src].</span>", \
-					"<span class='userdanger'>You trigger \the [src]!</span>")
+			if(silent_arming)
```

Comment:
Shouldn't this be `!silent_arming`? (if not silent, show message)

Reply:
Yup